src/main.ml

```diff
@@ -236,8 +256,19 @@ let warning_PatternNotExhaustive err errLines =
   }
 
 let warning_PatternUnused err errLines = raise Not_found
-let warning_OptionalArgumentNotErased err errLines = raise Not_found
-
+let warning_OptionalArgumentNotErased err errLines =
```

Comment:
This one isn't good. Did you forget the regex? It'll catch all the errors/warnings.